lib/main.coffee

```diff
@@ -2,26 +2,33 @@
 {find} = helpers = require 'atom-linter'
 path = require 'path'
 globule = require 'globule'
+ChildProcess = require('child_process')
+prefixPath = null
+fixPath = require 'fix-path'
```

Comment:
Do not use `fix-path` as it modifies `process.env.PATH` across the entire process, use [`consistent-path`](https://github.com/steelbrain/consistent-path) instead